testsuite/features/secondary/srv_retracted_patches.feature

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) 2015-2021 SUSE LLC
```

Comment:
Please rename this file starting with `min_`  as you use `sle_minion` to test.

Copyright: only `2021`, it's new

Reply:
Ok, my reasoning was that the feature is independent of the client stack (I only happened to use minion for all cases). But ok. I'm renaming.